src/hotspot/cpu/riscv/assembler_riscv.hpp

```diff
@@ -3185,125 +3259,95 @@ enum Nf {
-#define INSN(NAME)                                                                           \
-  void NAME(FloatRegister Rd, Register Rs, const int32_t offset) {                           \
-    /* fld -> c.fldsp/c.fld */                                                               \
-    if (do_compress()) {                                                                     \
-      if (is_c_fldsdsp(Rs, offset)) {                                                        \
-        c_fldsp(Rd, offset);                                                                 \
-        return;                                                                              \
-      } else if (is_c_fldsd(Rs, Rd, offset)) {                                               \
-        c_fld(Rd, Rs, offset);                                                               \
-        return;                                                                              \
-      }                                                                                      \
-    }                                                                                        \
-    _fld(Rd, Rs, offset);                                                                    \
+  void fld(FloatRegister Rd, Register Rs, const int32_t offset) {
+    /* fld -> c.fldsp/c.fld */
+    if (do_compress()) {
+      if (is_c_fldsdsp(Rs, offset)) {
+        c_fldsp(Rd, offset);
+        return;
+      } else if (is_c_fldsd(Rs, Rd, offset)) {
+        c_fld(Rd, Rs, offset);
+        return;
+      }
+    }
+    _fld(Rd, Rs, offset);
   }
 
-  INSN(fld);
-
-#undef INSN
-
 // --------------------------
-#define INSN(NAME)                                                                           \
-  void NAME(Register Rd, Register Rs, const int32_t offset) {                                \
-    /* sd -> c.sdsp/c.sd */                                                                  \
-    if (do_compress()) {                                                                     \
-      if (is_c_ldsdsp(Rs, Rd, offset, false)) {                                              \
-        c_sdsp(Rd, offset);                                                                  \
-        return;                                                                              \
-      } else if (is_c_ldsd(Rs, Rd, offset)) {                                                \
-        c_sd(Rd, Rs, offset);                                                                \
-        return;                                                                              \
-      }                                                                                      \
-    }                                                                                        \
-    _sd(Rd, Rs, offset);                                                                     \
+  void sd(Register Rd, Register Rs, const int32_t offset) {
```

Comment:
We can rename `Rd`/`Rs` to `Rs2`/`Rs1` to be more consistent with the specification.

Reply:
Fixed